lib/metasploit/framework/login_scanner/http.rb

```diff
@@ -396,6 +403,19 @@ def normalize_uri(target_uri)
           (self.uri.to_s + "/" + target_uri.to_s).gsub(/\/+/, '/')
         end
 
+        private
+
+        def validate_http_codes
+          if http_success_codes.nil?
+            @http_success_codes = DEFAULT_HTTP_SUCCESS_CODES
+          else
```

Comment:
Was it possible to use the `set_sane_defaults` pattern for this? 👀

Reply:
Updated 👍